src/hw_cbmc_irep_ids.h

```diff
@@ -9,7 +9,9 @@ IREP_ID_ONE(EX)
 IREP_ID_ONE(EU)
 IREP_ID_ONE(ER)
 IREP_ID_ONE(U)
+IREP_ID_ONE(weak_U)
 IREP_ID_ONE(R)
+IREP_ID_ONE(strong_R)
```

Comment:
Where is that used?

Reply:
You'd get that by negating weak until, and we don't have a frontend that generates it. Will drop.